test/components/plots/stackedPlotTests.ts

```diff
@@ -267,6 +267,52 @@ describe("Plots", () => {
     });
   });
 
+  describe("fail safe tests", () => {
+
+    var svg: D3.Selection;
+    var xScale: Plottable.Scale.Category;
+    var yScale: Plottable.Scale.Linear;
+    var data1: Object[];
+    var data2: Object[];
+
+    beforeEach(() => {
+      svg = generateSVG(600, 400);
+
+      data1 = [
+          { x: "A", y: "s", fill: "blue" },
+      ];
+      data2 = [
+          { x: "A", y: 1, fill: "red"},
+      ];
+      xScale = new Plottable.Scale.Category();
+      yScale = new Plottable.Scale.Linear();
+    });
+
+    afterEach(() => {
+      svg.remove();
+    });
+
+    it("conversion fails should be silent in Plot.StackedBar", () => {
+      var plot = new Plottable.Plot.StackedBar(xScale, yScale);
+      plot.addDataset("d1", data1);
+      plot.addDataset("d2", data2);
+      plot.project("fill", "fill");
+      plot.project("x", "x", xScale).project("y", "y", yScale);
+
+      var ds1PlotMetadata = <Plottable.Plot.StackedPlotMetadata>(<any> plot)._key2PlotDatasetKey.get("d1").plotMetadata;
+      var ds2PlotMetadata = <Plottable.Plot.StackedPlotMetadata>(<any> plot)._key2PlotDatasetKey.get("d2").plotMetadata;
+      var ds1FirstColumnOffset = ds1PlotMetadata.offsets.get("A");
+      var ds2FirstColumnOffset = ds2PlotMetadata.offsets.get("A");
+
+      assert.strictEqual(typeof ds1FirstColumnOffset, "number", "ds1 offset should be a valid number");
+      assert.strictEqual(typeof ds2FirstColumnOffset, "number", "ds2 offset should be a valid number");
+
+      assert.isFalse(Plottable._Util.Methods.isNaN(ds1PlotMetadata.offsets.get("A")), "ds1 offset should be a valid number");
```

Comment:
This `assert` has the same message as the one on Line 307:
* What's the difference between the two `assert`s?
* The message for this one should more accurately reflect what is being asserted. In this case, though, we should probably check that the value in question is a `number` and *also* that it is not `NaN`.